### PR TITLE
Lucia work

### PR DESCRIPTION
**Pull request by Tiseanu**
Updating VPN functionality

Test URLs:
- Before: https://main--helix-poc--enake.hlx.page/consumer/en/new/last-offer
- After: https://header-block-update--helix-poc--enake.hlx.page/consumer/en/new/last-offer

- Before: https://main--helix-poc--enake.hlx.page/consumer/en/new/last-offer
- After: https://header-block-update--helix-poc--enake.hlx.page/consumer/en/new/last-offer

- Before: https://main--helix-poc--enake.hlx.page/consumer/en/new/ultimate-flv1
- After: https://header-block-update--helix-poc--enake.hlx.page/consumer/en/new/ultimate-flv1

### CODE REVIEW

**Review by enake — APPROVED** [review]

File: blocks/c-device-protection-box/c-device-protection-box.css [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 .c-device-protection-box-container img {
     margin-left: -14em;
-    width: 155%;
+    width: 136%;
```
Comment (vladalexandruradulescu) [inline blocks/c-device-protection-box/c-device-protection-box.css:21]: why 136% ? and not 138 ? this is to specific ... maybe adjust it to a normal value

File: blocks/c-dropdownbox/c-dropdownbox.css [hunk 2]
```diff
@@ -144,6 +144,14 @@
     padding: 0 15px 5px;
   }
 
+  .c-dropdownbox-container p {
+    padding: 0 10px;
+  }
+
+  .c-dropdownbox-container .block .bottomText {
```
Comment (vladalexandruradulescu) [inline blocks/c-dropdownbox/c-dropdownbox.css:151]: . bottomText => .bottom-text

File: blocks/c-productswithvpn/c-productswithvpn.css [hunk 3]
```diff
@@ -277,11 +279,41 @@
     padding: 9px 0;
 }
 
+.c-productswithvpn-container .top_title h1, .c-productswithvpn-container .top_title h2 {
+    font-size: 28px;
+    width: 74%;
```
Comment (vladalexandruradulescu) [inline blocks/c-productswithvpn/c-productswithvpn.css:284]: 74% ... to specific, reduce it to 60% or a human readable value

File: blocks/c-productswithvpn/c-productswithvpn.css [hunk 4]
```diff
@@ -277,11 +279,41 @@
...
+    margin: 1em auto;
+    display: inline-block;
+    text-align: center;
+}
+
+.c-productswithvpn-container .top_title.p_left h1, .c-productswithvpn-container .top_title.p_left h2 {
+    width: 610px;
```
Comment (vladalexandruradulescu) [inline blocks/c-productswithvpn/c-productswithvpn.css:291]: above you are using % value for the width ( 74%, here 610px, maybe use uniform values everywhere ? )

File: blocks/c-productswithvpn/c-productswithvpn.css [hunk 5]
```diff
@@ -314,6 +346,8 @@
 @media (max-width: 768px) {
     .c-productswithvpn {
         display: block;
+        width: 92%;
```
Comment (vladalexandruradulescu) [inline blocks/c-productswithvpn/c-productswithvpn.css:349]: 92% to specific, use 60 / 70 / 80 / 90

File: blocks/c-productswithvpn/c-productswithvpn.css [hunk 6]
```diff
@@ -328,6 +362,13 @@
         line-height: 23px;
     }
 
+    .c-productswithvpn-container .top_title.p_left h1, .c-productswithvpn-container .top_title.p_left h2 {
+        width: 97%;
```
Comment (vladalexandruradulescu) [inline blocks/c-productswithvpn/c-productswithvpn.css:366]: same here

File: blocks/c-productswithvpn/c-productswithvpn.js [hunk 7]
```diff
@@ -27,12 +27,41 @@ export default function decorate(block) {
...
+    bulinaText,
+  } = metaData;
   const productsAsList = products.split(',');
 
   if (productsAsList.length) {
+    /// ///////////////////////////////////////////////////////////////////////
+    // set the title
+    if (typeof title !== 'undefined') {
+      const divTagTitle = document.createElement('div');
+      /*
```
Comment (vladalexandruradulescu) [inline blocks/c-productswithvpn/c-productswithvpn.js:44]: remove unused code

File: blocks/c-productswithvpn2/c-productswithvpn2.css [hunk 8]
```diff
@@ -405,38 +414,46 @@
...
         min-height: auto;
     }
 
     .c-productswithvpn2-container.has4boxes .c-productswithvpn2 > div {
-        width: 100%!important;
-        padding: 3%!important;
-        margin-bottom: 1em!important;
-        padding-bottom: 10px!important;
+        width: 100%;
+        padding: 3%;
```
Comment (vladalexandruradulescu) [inline blocks/c-productswithvpn2/c-productswithvpn2.css:426]: padding: 3% => use px value

File: blocks/c-productswithvpn2/c-productswithvpn2.css [hunk 9]
```diff
@@ -405,38 +414,46 @@
...
     .c-productswithvpn2 {
         display: block;
     }
 
     .c-productswithvpn2 > div {
-        width: 100%!important;
-        padding: 3%!important;
-        margin-bottom: 1em!important;
-        padding-bottom: 10px!important;
+        width: 94%;
```
Comment (vladalexandruradulescu) [inline blocks/c-productswithvpn2/c-productswithvpn2.css:442]: 94%

File: blocks/c-productswithvpn2/c-productswithvpn2.css [hunk 10]
```diff
@@ -405,38 +414,46 @@
...
+        padding: 3%;
```
Comment (vladalexandruradulescu) [inline blocks/c-productswithvpn2/c-productswithvpn2.css:443]: 3%

File: scripts/scripts.js [hunk 11]
```diff
@@ -998,6 +872,22 @@ const loadPage = async () => {
...
     item.parentElement.id = `${getIdentity}-${idx + 1}`;
   });
+
+  // addEventListener on VPN checkboxes
+  if (document.querySelector('.checkboxVPN')) {
+    document.querySelectorAll('.checkboxVPN').forEach((item) => {
+      item.addEventListener('click', (e) => {
+        const checkboxId = e.target.getAttribute('id');
+        /*
+        const checkboxClass = e.target.classList[0];
```
Comment (vladalexandruradulescu) [inline scripts/scripts.js:882]: remove unused code

File: styles/styles.css [hunk 12]
```diff
@@ -646,10 +646,19 @@ main .section.highlight {
     border-bottom: 1px solid white;
   }
 
+  .py-5 {
```
Comment (vladalexandruradulescu) [inline styles/styles.css:649]: we shouldn't override bootstrap classes

File: blocks/c-device-protection-box/c-device-protection-box.css [hunk 13]
```diff
@@ -18,11 +18,10 @@
 
 .c-device-protection-box-container img {
     margin-left: -14em;
-    width: 155%;
+    width: 120%;
```
Comment (vladalexandruradulescu) [inline blocks/c-device-protection-box/c-device-protection-box.css:21]: we must fix this container issues and the one from above

**Review by vladalexandruradulescu — CHANGES_REQUESTED** [review]

**Pull request merged**